dsl_parser/interfaces/utils.py

```diff
@@ -72,7 +74,7 @@ def merge_schema_and_instance_inputs(schema_inputs,
         flattened_schema_inputs.items() +
         instance_inputs.items())
 
-    validate_missing_inputs(merged_inputs)
+    validate_missing_inputs(merged_inputs, schema_inputs)
```

Comment:
I used `inputs_schema` for consistency with the `validate_inputs_types` function (the very next function, right next to `validate_missing_inputs`).
It seems that `schema_inputs` makes more sense - I'll also rename the argument to `validate_inputs_types` if that's ok?